pages/advanced/smart-account-modules/smart-account-modules-tutorial.mdx

```diff
@@ -0,0 +1,964 @@
+import { Callout } from 'nextra/components'
+
+# How to build an app with Safe and Safe Module
+
+In this tutorial, you will build an app that can:
```

Comment:
```suggestion
In this tutorial, you will:
```

the ending doesn't go well with the points. "build an app that can build a safe module"